nebullvm/tools/utils.py

```diff
@@ -214,23 +214,34 @@ def is_dict_type(data_sample: Any):
 def check_device(device: Optional[str]) -> Device:
     if device is None:
         if gpu_is_available():
-            device = Device.GPU
+            device = Device(DeviceType.GPU)
         else:
-            device = Device.CPU
+            device = Device(DeviceType.CPU)
     else:
-        if device.lower() == "gpu":
+        if device.lower() == "gpu" or "cuda" in device.lower():
+            if device.lower() == "gpu":
+                logger.warning(
+                    "gpu tag is deprecated, please use cuda or cuda:0 instead."
+                )
```

Comment:
should we deprecate it? "cuda" is quite obvious for torch users. However for TF and ONNX ones it could be better to keep the "gpu" device. WDYT?

Reply:
Yep I agree, we can keep it